### PR TITLE
Add Timestamp + update Diagnosis and profile picture

### PR DESCRIPTION
**Pull request by omegafishy**
Add Timestamp, update Diagnosis to include Timestamp + doctor name field, update profile picture

### CODE REVIEW

File: src/main/java/seedu/address/model/medicalhistory/Diagnosis.java [hunk 1]
```diff
@@ -11,27 +12,46 @@
...
+        this.doctorInCharge = doctorName;
+        this.timestamp = new Timestamp();
     }
 
     /**
      * Constructs a {@code Diagnosis} from another {@code Diagnosis}
      *
      * @param d A valid diagnosis.
-     */
+     *///todo is it required to make diagnosis immutable?
```
Comment (jeffypie369) [inline src/main/java/seedu/address/model/medicalhistory/Diagnosis.java]: I think it should be

Comment (jeffypie369) [inline src/main/java/seedu/address/model/medicalhistory/Diagnosis.java]: I think it should be

File: src/main/java/seedu/address/model/medicalhistory/Diagnosis.java [hunk 2]
```diff
@@ -43,6 +63,15 @@ public static boolean isValidDiagnosis(String test) {
         return true;
     }
 
+    /**
+     * Returns true if a given string is a valid doctor's name.
+     */
+    public static boolean isValidDoctor(String test) {
+        //todo complete regex
```
Comment (jeffypie369) [inline src/main/java/seedu/address/model/medicalhistory/Diagnosis.java]: Remember to update

File: src/main/java/seedu/address/model/medicalhistory/Timestamp.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
+package seedu.address.model.medicalhistory;
```
Comment (jeffypie369) [inline src/main/java/seedu/address/model/medicalhistory/Timestamp.java]: Add Test for this class

File: src/main/java/seedu/address/model/medicalhistory/Diagnosis.java (hunk 2 above)
Comment (omegafishy) [reply src/main/java/seedu/address/model/medicalhistory/Diagnosis.java]: ok

**Pull request merged**